src/exercise/01.js

```diff
@@ -1,18 +1,27 @@
 // useReducer: simple Counter
 // http://localhost:3000/isolated/exercise/01.js
 
-import * as React from 'react'
+import React, { useReducer } from 'react';
+
+function countReducer(state, {type, step}) {
+  switch (type) {
+    case 'INCREMENT':
+      return { ...state, 'count': state.count + step };
+
+    default:
```

Comment:
Good call on this, forgot the `default` is usually just to return whatever the current state is in a reducer (I made the default `INCREMENT` in my video)!